timesys/high-level-triggers.py

```diff
@@ -662,6 +662,20 @@
             'BO-49D:PS-CH:BCKPLN', 'BO-49U:PS-CV:BCKPLN',
             ),
         },
+    'BO-Glob:TI-LLRF-FDL': {
```

Comment:
put this trigger definition close to other RF triggers.

Why this trigger needs to listen to the Cycle event?

Reply:
Hi @fernandohds564 , I didn't understand your point about the Cycle event.

Ok, I can move the definition

Reply:
Never mind, the GitHub app interface got me confused. I saw the SRC definition of the other trigger.